lib/index.js

```diff
@@ -48,10 +48,14 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const bodyParser = require('body-parser')
   const app = express()
   app.use(require('compression')())
   app.use(require('cors')())
-  app.use(require('body-parser').json())
+  app.use(bodyParser.json({limit:'50mb'}))
+  app.use(bodyParser.urlencoded({
```

Comment:
Why is urlencoded needed to solve the size limit?

Reply:
Updated after testing. Not needed as you suspected.